### PR TITLE
Update certificates template to use publicIP from SG

### PR DESCRIPTION
**Pull request by stgleb**

### CODE REVIEW

File: templates/certificates.tpl [hunk 1]
```diff
@@ -3,7 +3,7 @@ KUBERNETES_SSL_DIR={{ .KubernetesConfigDir }}/ssl
 sudo mkdir -p ${KUBERNETES_SSL_DIR}
 
 #sleep 10
-#FLANNELIP=$(ifconfig flannel.1|grep 'inet addr'| awk '{print substr($2,6,12)}')
+FLANNELIP=$(ifconfig flannel.1|grep 'inet addr'| awk '{print substr($2,6,12)}')
```
Comment (egopher) [inline templates/certificates.tpl]: What if there is no flannel.1 interface?

File: pkg/workflows/steps/certificates/certificates.go [hunk 2]
```diff
@@ -34,8 +35,12 @@ func New(script *template.Template) *Step {
 }
 
 func (s *Step) Run(ctx context.Context, out io.Writer, config *steps.Config) error {
-	config.CertificatesConfig.MasterHost = config.GetMaster().PrivateIp
-
+	if master := config.GetMaster(); master != nil {
```
Comment (saheienko) [inline pkg/workflows/steps/certificates/certificates.go]: What master would be selected for the worker node?

File: templates/certificates.tpl [hunk 3]
```diff
@@ -19,9 +19,10 @@ DNS.1 = kubernetes
 DNS.2 = kubernetes.default
 DNS.3 = kubernetes.default.svc
 DNS.4 = kubernetes.default.svc.cluster
-IP.1 = {MASTER_HOST}
-IP.2 = {PRIVATE_HOST}
-IP.3 = 10.3.0.1
+IP.1 = {FLANNELIP}
```
Comment (saheienko) [inline templates/certificates.tpl]: How does it used?

File: templates/certificates.tpl [hunk 4]
```diff
@@ -19,9 +19,10 @@ DNS.1 = kubernetes
...
+IP.2 = {MASTER_HOST}
+IP.3 = {PRIVATE_HOST}
```
Comment (saheienko) [inline templates/certificates.tpl]: You can set addresses here without using sed.

File: pkg/workflows/steps/certificates/certificates.go (hunk 2 above)
Comment (stgleb) [reply pkg/workflows/steps/certificates/certificates.go]: Any active master node @saheienko

File: templates/certificates.tpl (hunk 1 above)
Comment (stgleb) [reply templates/certificates.tpl]: Removed that, that is obsolete

File: pkg/workflows/steps/certificates/certificates.go (hunk 2 above)
Comment (stgleb) [reply pkg/workflows/steps/certificates/certificates.go]: Either current node we are working on if it is master or any random active master https://github.com/supergiant/supergiant/blob/master/pkg/workflows/steps/config.go#L353

File: templates/certificates.tpl (hunk 3 above)
Comment (stgleb) [reply templates/certificates.tpl]: Removed

**Pull request merged**